src/components/MovieInfo/MovieInfo.js

```diff
@@ -40,15 +44,35 @@ class MovieInfo extends Component {
         }}>
           <img src={this.state.currentMovie.poster_path} alt='movie poster' className='ind-movie-poster' />
           <article className='movie-stats'>
-            <h2 className='title'>{this.state.currentMovie.title}</h2>
-            <h3 className='avg-rating'>Rating: {this.roundRating(this.state.currentMovie.average_rating)}</h3>
-            <p className='release-date'>Release Date: {this.state.currentMovie.release_date}</p>
-            <p className='overview'>Overview: {this.state.currentMovie.overview}</p>
-            <p className='genres'>Genres: {this.state.currentMovie.genres}</p>
-            <p className='budget'>Budget: ${this.state.currentMovie.budget}</p>
-            <p className='revenue'>Revenue: ${this.state.currentMovie.revenue}</p>
-            <p className='runtime'>Runtime: {this.state.currentMovie.runtime}</p>
-            <p className='tagline'>Tagline: {this.state.currentMovie.tagline}</p>
+            <div className='title-rating-genres'>
```

Comment:
Look at this overhaul!  I can't tell you how amazing this all looks, phenomenal work my friend!